node_modules/oae-zencoder/lib/init.js

```diff
@@ -13,28 +13,38 @@
  * permissions and limitations under the License.
  */
 
-var log = require('oae-logger').logger('oae-mediacore-init');
-
-var MediaCoreProcessor = require('./processor');
-
+var _ = require('underscore');
 var Cassandra = require('oae-util/lib/cassandra');
+var log = require('oae-logger').logger('oae-zencoder-init');
 var PreviewProcessorAPI = require('oae-preview-processor');
 
+var ZencoderAPI = require('./api');
+var ZencoderProcessor = require('./processor');
+
 module.exports = function(config, callback) {
-    // Register the mediacore video processor
-    PreviewProcessorAPI.registerProcessor('oae-file-mediacore', MediaCoreProcessor);
+    var _config = config.zencoder;
+    _.extend(_config, {'amazons3-bucket': config.storage['amazons3-bucket'],
+                       'amazons3-region': config.storage['amazons3-region'],
+                       'amazons3-access-key': config.storage['amazons3-access-key'],
+                       'amazons3-secret-key': config.storage['amazons3-secret-key']
+            });
```

Comment:
That said, taking storage configuration from one config location and just overwriting siblings on another config object doesn't seem very straightforward. I think if we make the storage configuration scoped in the zencoder config, then it feels like it's more what we're trying to do here.